dotcom-rendering/scripts/json-schema/check-schema.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const fs = require('fs');
+
+const root = path.resolve(__dirname, '..', '..');
+const { generateArticleSchema, generateFrontSchema } = require('./schema');
+
+const existingArticleSchema = fs.readFileSync(
+	`${root}/src/model/json-schema.json`,
```

Comment:
As part of this refactor, we could rename this to `article-schema`?

Reply:
I'd actually be tempted to do this in a second PR, and keep this PR solely about the github action!

Reply:
> solely about the github action

That makes sense, but we have done quite a big refactor on gen-schema already... smaller is better though.